ui/app/models/cluster.js

```diff
@@ -75,9 +75,9 @@ export default DS.Model.extend({
     const glyph = 'check-circle-outline';
 
     const glyphs = {
-      'stream-wals': 'android-sync',
+      'stream-wals': 'check-circle-outline',
       'merkle-diff': 'android-sync',
-      'merkle-sync': null,
+      'merkle-sync': 'android-sync',
```

Comment:
thanks for doing this! this paves the way for the next set of improvements to the status menu (showing error icons if the cluster is in a bad state).